spec/system/email_change_spec.rb

```diff
@@ -120,9 +120,8 @@ def get_link_from_email(type)
 
     find(".confirm-new-email .btn-primary").click
     find(".second-factor-token-input").fill_in with: second_factor.totp_object.now
-    find("button[type=submit]").click
+    find("button[type=submit]:not([disabled])").click
 
-    expect(page).to have_content(I18n.t("js.second_factor_auth.redirect_after_success"))
```

Comment:
Removing this assertion because this is a very transient message. The way we have written our code is to redirect the user immediately after flashing this message. There is no way to reliable test that this message is shown unless we redesign how or when the message is displayed.

Reply:
Sounds like a pretty bad design. If the test struggles to see it, so will users? 😅